sormas-app/app/src/main/res/layout/fragment_case_edit_layout.xml

```diff
@@ -785,6 +821,79 @@
                         style="@style/ControlSecondOfTwoColumnsStyle" />
                 </LinearLayout>
 
+                <LinearLayout
```

Comment:
This has nothing to do with your ticket, please do a proper merge with the development branch